js/views/autocomplete.es6.js

```diff
@@ -0,0 +1,38 @@
+const Parent = window.DDG.base.View;
+
+function Autocomplete (ops) {
+
+    this.model = ops.model;
+    this.pageView = ops.pageView;
+    this.template = ops.template;
+
+    Parent.call(this, ops);
+
+    this.bindEvents([
+      [this.model.store.subscribe, 'change:search', this._handleSearchTextUpdate],
```

Comment:
This is how you subscribe to another model's changes. Here, it is `change:search` because `search` is the `this.modelName` that belongs to the model we are listening to.